codegen/templates/endpoint.tmpl

```diff
@@ -147,10 +160,10 @@ func (w {{$workflow}}) Handle(
 	}
 
 	{{if eq .ResponseType "" -}}
-	return nil, nil
+	return respHeaders, nil
```

Comment:
We do not want to proxy all response headers. we need a response header whitelist.

The request header proxy whitelist & response header proxy whitelist should be two seperate configs in the endpoint json config.